data/human/events.txt

```diff
@@ -1167,12 +1167,95 @@ event "navy occupying the south"
 event "catalytic ramscoop available"
 	outfitter "Deep Sky Basics"
 		"Catalytic Ramscoop"
-
-
-
-event "fw gets catalytic ramscoop"
-	outfitter "Delta V Advanced"
-		"Catalytic Ramscoop"
+	fleet "Republic Logistics"
+		add variant 19
+			"Auxiliary (Mark II)"
+			"Dropship" 2
+			"Lance" 2
+			"Cruiser"
+			"Combat Drone" 4
+			"Frigate"
+			"Gunboat" 2
+		add variant 19
+			"Auxiliary (Cargo Mark II)"
+			"Dropship" 2
+			"Lance" 2
+			"Cruiser"
+			"Combat Drone" 4
+			"Frigate"
+			"Gunboat" 2
+		add variant 19
+			"Auxiliary (Transport Mark II)"
+			"Dropship" 2
+			"Lance" 2
+			"Cruiser"
+			"Combat Drone" 4
+			"Frigate"
+			"Gunboat" 2
+		add variant 1
+			"Auxiliary (Cargo Mark II)" 3
+			"Dropship" 3
+			"Lance" 9
+			"Cruiser" 2
+			"Combat Drone" 8
+		add variant 1
+			"Auxiliary (Transport Mark II)"
+			"Auxiliary (Cargo Mark II)"
+			"Auxiliary (Mark II)"
+			"Dropship" 6
+			"Lance" 6
+			"Cruiser" 2
+			"Combat Drone" 8
+		add variant 1
+			"Auxiliary (Transport Mark II)" 3
+			"Dropship" 9
+			"Lance" 3
+			"Cruiser" 2
+			"Combat Drone" 8
+		remove variant
```

Comment:
The other event that updates Navy fleets doesn't remove variants.

Reply:
Due to economies of scale and standardisation, having these fully replace the old variants would reduce maintenance costs enough while adding such a large direct benefit to long range logistics, it would be justified to fully replace them instead of simply supplementing them, in my eyes. It is also a very small change as opposed to a nearly complete one, with the other Mark IIs.

I can change it so that it still only makes them as common as the old variants as opposed to replacing them, but, again, it's such a small and direct upgrade i don't see why to do that. This is my thought process behind this being different.

Reply:
I'd rather avoid overcomplicating things, but if you think that the extra overhead is worth it, go ahead.